### ISSUE TITLE
column "exemplar_label_values" does not exist

### ISSUE THREAD

**Issue by Yamakaky** [timescale/promscale#1567]
I have an issue when ingesting metrics with exemplars. Unless I'm mistaken, I do not get exemplars when doing promql queries. What additional info would you need?

```
2022-08-19 12:44:02 UTC [85]: [62ff83ec.55-508] postgres@tsdb,app=promscale@0.13.0 [23505] ERROR:  duplicate key value violates unique constraint "ei_latency_bucket"
2022-08-19 12:44:02 UTC [85]: [62ff83ec.55-509] postgres@tsdb,app=promscale@0.13.0 [23505] DETAIL:  Key (series_id, "time")=(17372, 2022-08-19 12:44:02.235+00) already exists.
2022-08-19 12:44:02 UTC [85]: [62ff83ec.55-510] postgres@tsdb,app=promscale@0.13.0 [23505] CONTEXT:  COPY latency_bucket, line 7
2022-08-19 12:44:02 UTC [85]: [62ff83ec.55-511] postgres@tsdb,app=promscale@0.13.0 [23505] STATEMENT:  copy "prom_data_exemplar"."latency_bucket" ( "time", "series_id", "exemplar_label_values", "value" ) from stdin binary;
2022-08-19 12:44:02 UTC [85]: [62ff83ec.55-512] postgres@tsdb,app=promscale@0.13.0 [42703] ERROR:  column "exemplar_label_values" does not exist at character 29
2022-08-19 12:44:02 UTC [85]: [62ff83ec.55-513] postgres@tsdb,app=promscale@0.13.0 [42703] STATEMENT:  select "time", "series_id", "exemplar_label_values", "value" from "s1_latency_bucket"
2022-08-19 12:44:02 UTC [85]: [62ff83ec.55-514] postgres@tsdb,app=promscale@0.13.0 [42703] ERROR:  column "exemplar_label_values" does not exist at character 29
2022-08-19 12:44:02 UTC [85]: [62ff83ec.55-515] postgres@tsdb,app=promscale@0.13.0 [42703] STATEMENT:  select "time", "series_id", "exemplar_label_values", "value" from "s0_latency_bucket"
2022-08-19 12:44:03 UTC [85]: [62ff83ec.55-516] postgres@tsdb,app=promscale@0.13.0 [23505] ERROR:  duplicate key value violates unique constraint "ei_latency_bucket"
2022-08-19 12:44:03 UTC [85]: [62ff83ec.55-517] postgres@tsdb,app=promscale@0.13.0 [23505] DETAIL:  Key (series_id, "time")=(9565, 2022-08-19 12:44:03.236+00) already exists.
2022-08-19 12:44:03 UTC [85]: [62ff83ec.55-518] postgres@tsdb,app=promscale@0.13.0 [23505] CONTEXT:  COPY latency_bucket, line 2
2022-08-19 12:44:03 UTC [85]: [62ff83ec.55-519] postgres@tsdb,app=promscale@0.13.0 [23505] STATEMENT:  copy "prom_data_exemplar"."latency_bucket" ( "time", "series_id", "exemplar_label_values", "value" ) from stdin binary;
2022-08-19 12:44:03 UTC [85]: [62ff83ec.55-520] postgres@tsdb,app=promscale@0.13.0 [42703] ERROR:  column "exemplar_label_values" does not exist at character 29
2022-08-19 12:44:03 UTC [85]: [62ff83ec.55-521] postgres@tsdb,app=promscale@0.13.0 [42703] STATEMENT:  select "time", "series_id", "exemplar_label_values", "value" from "s1_latency_bucket"
2022-08-19 12:44:03 UTC [85]: [62ff83ec.55-522] postgres@tsdb,app=promscale@0.13.0 [42703] ERROR:  column "exemplar_label_values" does not exist at character 29
2022-08-19 12:44:03 UTC [85]: [62ff83ec.55-523] postgres@tsdb,app=promscale@0.13.0 [42703] STATEMENT:  select "time", "series_id", "exemplar_label_values", "value" from "s0_latency_bucket"
2022-08-19 12:44:05 UTC [85]: [62ff83ec.55-524] postgres@tsdb,app=promscale@0.13.0 [23505] ERROR:  duplicate key value violates unique constraint "ei_latency_bucket"
2022-08-19 12:44:05 UTC [85]: [62ff83ec.55-525] postgres@tsdb,app=promscale@0.13.0 [23505] DETAIL:  Key (series_id, "time")=(9791, 2022-08-19 12:44:05.254+00) already exists.
2022-08-19 12:44:05 UTC [85]: [62ff83ec.55-526] postgres@tsdb,app=promscale@0.13.0 [23505] CONTEXT:  COPY latency_bucket, line 13
2022-08-19 12:44:05 UTC [85]: [62ff83ec.55-527] postgres@tsdb,app=promscale@0.13.0 [23505] STATEMENT:  copy "prom_data_exemplar"."latency_bucket" ( "time", "series_id", "exemplar_label_values", "value" ) from stdin binary;
2022-08-19 12:44:05 UTC [85]: [62ff83ec.55-528] postgres@tsdb,app=promscale@0.13.0 [42703] ERROR:  column "exemplar_label_values" does not exist at character 29
2022-08-19 12:44:05 UTC [85]: [62ff83ec.55-529] postgres@tsdb,app=promscale@0.13.0 [42703] STATEMENT:  select "time", "series_id", "exemplar_label_values", "value" from "s1_latency_bucket"
2022-08-19 12:44:05 UTC [85]: [62ff83ec.55-530] postgres@tsdb,app=promscale@0.13.0 [42703] ERROR:  column "exemplar_label_values" does not exist at character 29
2022-08-19 12:44:05 UTC [85]: [62ff83ec.55-531] postgres@tsdb,app=promscale@0.13.0 [42703] STATEMENT:  select "time", "series_id", "exemplar_label_values", "value" from "s0_latency_bucket"
```

```
timescaledb 2022-08-19 12:41:50 UTC [80]: [62ff83eb.50-324] postgres@tsdb,app=promscale@0.13.0 [23505] ERROR:  duplicate key 
```

```
tsdb=# select version();
                                                           version                                                            
------------------------------------------------------------------------------------------------------------------------------
 PostgreSQL 14.5 (Ubuntu 14.5-1.pgdg22.04+1) on x86_64-pc-linux-gnu, compiled by gcc (Ubuntu 11.2.0-19ubuntu1) 11.2.0, 64-bit

tsdb=# SELECT extname, extversion FROM pg_extension;
       extname       | extversion 
---------------------+------------
 plpgsql             | 1.0
 timescaledb         | 2.7.2
 promscale           | 0.5.4
 timescaledb_toolkit | 1.8.0
```

**Comment by cevian** [comment]
@Harkishen-Singh please take a look at this

**Comment by Harkishen-Singh** [comment]
I can confirm that this is broken. I do not see any exemplar metric ingested into TimescaleDB, as `select * from _prom_catalog.exemplar;` gives me an empty result. Will investigate into this.

**Comment by Harkishen-Singh** [comment]
Update: I can query exemplars in Grafana (screenshot) generated using [prom-generator](https://github.com/harkishen-singh/prom-generator), but I do see `ERROR: column \"exemplar_label_values\" does not exist (SQLSTATE 42703)` in Promscale logs and the exemplar table in Promscale does not populate past 1 exemplar.


<img width="1824" alt="image" src="https://user-images.githubusercontent.com/33792202/185905893-84d7d6c6-e80f-4762-af1f-7e48f391084d.png">

**Comment by Harkishen-Singh** [comment]
@Yamakaky , this error should be fixed after #1574 is merged. We plan to release `0.14.0` sometime next week and that release should have this fix.

**Comment by Yamakaky** [comment]
Nice, I'll try that when it's out!
